app/src/components/main/modal/modalLayout/modalField/modalField.jsx

```diff
@@ -28,6 +30,8 @@ export const ModalField = ({
   labelWidth,
   alignLeft,
   noMinHeight,
+  showInfoIcon,
```

Comment:
I think the `showInfoIcon` property is redundant because the condition of show tooltip is title existence. So I suggest to rename `iconTitle` to smth like `labelTip` and use it in the condition.